helm-chart/images/binderhub/binderhub_config.py

```diff
@@ -20,6 +20,10 @@ def get_config(key, default=None):
 c.BinderHub.debug = True
 
 c.BinderHub.docker_image_prefix = get_config('binder.registry.prefix')
+c.BinderHub.docker_registry_host = get_config('binder.registry.host')
```

Comment:
This is the source of the helm test failure. When registry is not enabled, these will be undefined and thus None, which produces an error. If you add `, ''` here and below for docker_token_url it will default to an empty string and be a safe value. You can also do the same with `if get_config` that you have for auth-host to only load the config if it's defined.